modules/nf-core/filtlong/tests/main.nf.test

```diff
@@ -0,0 +1,96 @@
+nextflow_process {
+
+    name "Test Process FILTLONG"
+    script "../main.nf"
+    process "FILTLONG"
+    config "./nextflow.config"
+    tag "filtlong"
+    tag "modules"
+    tag "modules_nfcore"
+
+    test("sarscov2 nanopore [fastq]") {
+
+        when {
+            params {
+                outdir = "$outputDir"
+            }
+            process {
+                """
+                input[0] = [
+                            [ id:'test', single_end:false ], // meta map
+                            [],
+                            [ file(params.test_data['sarscov2']['nanopore']['test_fastq_gz'], checkIfExists: true) ]
+                           ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert path(process.out.log.get(0).get(1)).readLines().contains("Scoring long reads")},
```

Comment:
This could be in the snapshot

Reply:
I'm sorry but I am confused here: the log is not stable between different profiles (due to how numbers are formatted, e.g. 32,313 versus 32313). That's why I assert the presence of a line. Do you mean that this assertion should be part of the snapshot?

Reply:
Yeah exactly! So we can just bump it if it updates.

That's a weird formatting thing to have between profiles 🤔